src/ensembl/src/content/app/custom-download/state/attributes/attributesActions.ts

```diff
@@ -25,65 +27,243 @@ export const setAttributes = createAsyncAction(
+    updateActiveConfigurationForGenome({
+      activeGenomeId,
+      data: {
+        ...set(
+          getCustomDownloadActiveGenomeConfiguration(getState()),
+          'attributes.content.orthologueAttributes.orthologues',
+          orthologues
+        )
+      }
+    })
+  );
+};
+
+export const setOrthologueShowBestMatches: ActionCreator<
+  ThunkAction<void, any, null, Action<string>>
+> = (showBestMatches: boolean) => (dispatch, getState: () => RootState) => {
+  const activeGenomeId = getCustomDownloadActiveGenomeId(getState());
+
+  if (!activeGenomeId) {
+    return;
+  }
+
+  dispatch(
+    updateActiveConfigurationForGenome({
+      activeGenomeId,
+      data: {
+        ...set(
+          getCustomDownloadActiveGenomeConfiguration(getState()),
+          'attributes.content.orthologue.showBestMatches',
+          showBestMatches
+        )
+      }
+    })
+  );
+};
+
+export const setOrthologueShowAll: ActionCreator<
+  ThunkAction<void, any, null, Action<string>>
+> = (showAll: boolean) => (dispatch, getState: () => RootState) => {
+  const activeGenomeId = getCustomDownloadActiveGenomeId(getState());
+
+  if (!activeGenomeId) {
+    return;
+  }
+
+  dispatch(
+    updateActiveConfigurationForGenome({
+      activeGenomeId,
+      data: {
+        ...set(
+          getCustomDownloadActiveGenomeConfiguration(getState()),
+          'attributes.content.orthologue.showAll',
+          showAll
+        )
+      }
+    })
+  );
+};
+
+export const setOrthologueApplyToAllSpecies: ActionCreator<
+  ThunkAction<void, any, null, Action<string>>
+> = (applyToAllSpecies: boolean) => (dispatch, getState: () => RootState) => {
+  const activeGenomeId = getCustomDownloadActiveGenomeId(getState());
+
+  if (!activeGenomeId) {
+    return;
+  }
+
+  dispatch(
+    updateActiveConfigurationForGenome({
+      activeGenomeId,
+      data: {
+        ...set(
+          getCustomDownloadActiveGenomeConfiguration(getState()),
+          'attributes.content.orthologue.applyToAllSpecies',
+          applyToAllSpecies
+        )
+      }
+    })
+  );
+};
+
+export const setOrthologueSearchTerm: ActionCreator<
+  ThunkAction<void, any, null, Action<string>>
+> = (searchTerm: string) => (dispatch, getState: () => RootState) => {
+  const activeGenomeId = getCustomDownloadActiveGenomeId(getState());
+
+  if (!activeGenomeId) {
+    return;
+  }
+
+  dispatch(
+    updateActiveConfigurationForGenome({
+      activeGenomeId,
+      data: {
+        ...set(
+          getCustomDownloadActiveGenomeConfiguration(getState()),
+          'attributes.content.orthologue.searchTerm',
+          searchTerm
+        )
+      }
+    })
+  );
+};
 
 export const setOrthologueSpecies = createAsyncAction(
   'custom-download/set-orthologue-species-request',
   'custom-download/set-orthologue-species-success',
   'custom-download/set-orthologue-species-failure'
-)<{ searchTerm: string }, CheckboxGridOption[], Error>();
+)<{ searchTerm: string }, undefined, Error>();
```

Comment:
not `void`?
(although I am not sure what the difference is)

Reply:
That's done